ci-scripts/yocto2.groovy

```diff
@@ -247,37 +210,65 @@ void buildManifest(String variantName, String imageName, String layerToReplace="
         boolean smokeTests = getBoolEnvVar("SMOKE_TEST", false)
         boolean bitbakeTests = getBoolEnvVar("BITBAKE_TEST", false)
         boolean yoctoCompatTest = getBoolEnvVar("YOCTO_COMPATIBILITY_TEST", false)
-        setupBitbake(yoctoDir, templateConf, doArchiveCache, smokeTests, analyzeImage)
-        setupCache(yoctoDir, yoctoCacheURL)
+        stage("Setup bitbake and cache") {
+            setupBitbake(yoctoDir, templateConf, doArchiveCache, smokeTests, analyzeImage)
+            setupCache(yoctoDir, yoctoCacheURL)
+        }
 
         // Build the images
         try {
+            stage("Bitbake ${imageName} for ${variantName}") {
+                buildImage(yoctoDir, imageName, variantName)
+            }
+
             boolean buildUpdate = variantName.startsWith("rpi") || variantName.startsWith("intel") || variantName.startsWith("arp")
-            buildImageAndSDK(yoctoDir, imageName, variantName, buildUpdate)
+            stage("Bitbake Update ${imageName} for ${variantName}") {
```

Comment:
"Update image" is misleading, consider to change to `stage("Bitbake cpio/swu archive")`.

Reply:
Bitbake cpio/swu archive for ${variantName} will be ok?

Reply:
Sure